Testing/functional/tests/cypress/integration/e2e/timeline/dependent.js

```diff
@@ -50,15 +50,20 @@ function enabledDatasetShouldBePresent(dataset) {
         Cypress.env("keycloak.username"),
         Cypress.env("keycloak.password"),
         AuthMethod.KeyCloak,
-        `/dependents/${authorizedDependentHdid}/timeline`
+        `/dependents/${
+            overrideDependentHdid ?? authorizedDependentHdid
+        }/timeline`
```

Comment:
Nit: it might be cleaner to assign this to a variable at the start of the function, e.g.

```
const hdid = overrideDependentHdid ?? authorizedDependentHdid;
```